### DISCUSSION TITLE
Workaround for MFA

### DISCUSSION THREAD

**Discussion by mcpeach** [balansse/homebridge-vivint#130] [Q&A]
Just got my Hoobs Box, and tried the workaround for Vivint listed - I enabled mfa from the command line, got the refreshToken and added it to the config.json.  However, I'm not seeing any accessories listed when adding the bridge to Home Kit.  Do I need to add these manually, or run something else to discover them?   The interface gives me the option to "unlink account" now, so I'm assuming it's actually connected to my account. 

{
    "accessories": [],
    "platforms": [
        {
            "platform": "Vivint",
            "refreshToken": "tokenValue"
        }
    ]
}

**Comment by ronster22** [comment]
I too am having authentication issues. Had to restart and update my HOOBS hub, also updated to 1.8.5 for the vivint bridge. Via the UI, after entering my credentials and access code, the UI doesn't take the code. Using the CLI, i can get a token...but no accessories are added...
